### PR TITLE
dfu-programmer: update version to 0.7.2

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description
- bump version to 0.7.2
- add license
- correctly align
- remove useless patch (now use pkg-config)
- move to nomaintainer

Closes: https://trac.macports.org/ticket/59849

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.15.2 19C57
Xcode 11.3 11C29

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: cross/dfu-programmer/Portfile [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-                  Atmel "FLIP" program for flashing devices does not support flashing via USB \
-                  on Linux, and because standard DFU loaders do not work for Atmel's chips.
-homepage          http://dfu-programmer.sourceforge.net/
-platforms         darwin
-master_sites      sourceforge
+name                dfu-programmer
+version             0.7.2
+categories          cross devel
+license             GPL-2
+maintainers         nomaintainer
```
Comment (mf2k) [inline cross/dfu-programmer/Portfile]: Does the maintainer agree to this? Was a port abandoned ticket filed?

Comment (ra1nb0w) [reply cross/dfu-programmer/Portfile]: was not updated by anyone including the maintainer for the last ~6 years; I remember a policy like this in the past. Am I wrong?

Comment (mf2k) [reply cross/dfu-programmer/Portfile]: The policy is that a port abandoned ticket must be filed before a maintainer can be removed. 

See [section 7.4.2](https://guide.macports.org/chunked/project.update-policies.html) of the guide.

Comment (ra1nb0w) [reply cross/dfu-programmer/Portfile]: Sorry for the mistake. Ticket [created](https://trac.macports.org/ticket/59849). Now we wait 72 hours.
Thank you

Comment (ra1nb0w) [reply cross/dfu-programmer/Portfile]: changed commit with `Closes` line

**Comment by ra1nb0w** [comment]
ping

**Comment by ra1nb0w** [comment]
thank you!

**Pull request merged**